### PR TITLE
Write code for TC07

### PR DESCRIPTION
**Pull request by lqthai07t3**
- Add more code to define Register tab in BasePage class.
- Add RegisterPage class and define element in Register page.
- Add RegisterTest class and code for TC07.
- Run code and check.

### CODE REVIEW

File: src/test/java/tests/RegisterTests.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    private RegisterPage registerPage = new RegisterPage();
+
+    @Test(testName = "User can create new account")
+    public void tc07_RegisterWithValidInformation(){
+        LogHelper.info("Go to Register Tab");
+        homePage.clickRegisterTab();
+
+        LogHelper.info("Register with valid information into all fields");
+        String newEmail = DataHelper.getRandomEmail();
+        String newPassword = DataHelper.getRandomPassword(8);
```
Comment (tanmle) [inline src/test/java/tests/RegisterTests.java:21]: again, you don't need to random password.

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/tests/RegisterTests.java (hunk 1 above)
Comment (lqthai07t3) [reply src/test/java/tests/RegisterTests.java]: Random Email needed for TC always works. So, I keep it
Random password and random pid have been removed.
Please check again

File: src/main/java/pages/RegisterPage.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class RegisterPage extends BasePage{
+    //Locators
+    private By txtEmail = By.id("email");
+    private By txtPassword = By.id("password");
+    private By txtConfirmPassword = By.id("confirmPassword");
+    private By txtPid = By.id("pid");
+    private By btnRegister = By.xpath("//input[@value='Register']");
```
Comment (MinhQuan119) [inline src/main/java/pages/RegisterPage.java:13]: Can this control catch by CSS selector?

Comment (lqthai07t3) [reply src/main/java/pages/RegisterPage.java:13]: Yeah, ofcouse.

Comment (lqthai07t3) [reply src/main/java/pages/RegisterPage.java]: I has fixed it. Please check again

File: src/main/java/pages/RegisterPage.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    //Methods
+    public void register(String email, String pass, String confirmPassword, String pid) {
+        getTxtEmail().sendKeys(email);
+        getTxtPassword().sendKeys(pass);
+        getTxtConfirmPassword().sendKeys(confirmPassword);
+        getTxtPid().sendKeys(pid);
+        getBtnRegister().click();
+    }
+
+    public String getContent(){ return getLblContent().getText(); }
```
Comment (tanmle) [inline src/main/java/pages/RegisterPage.java:33]: what is content?

Comment (lqthai07t3) [reply src/main/java/pages/RegisterPage.java]: Please check again

File: src/main/java/pages/RegisterPage.java (hunk 2 above)
Comment (MinhQuan119) [reply src/main/java/pages/RegisterPage.java]: It changed to By.cssSelector("p.form-actions>input").
I mean can you catch cssSelector with @value='Register'

Comment (lqthai07t3) [reply src/main/java/pages/RegisterPage.java]: Please check it again :)

**Review by tanmle — APPROVED** [review]

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**